src/test/java/com/rultor/agents/daemons/EndsDaemonITCase.java

```diff
@@ -103,6 +106,38 @@ public void readsExitCodeCorrectly() throws IOException {
     }
 
     /**
+     * EndsDaemon can deprecates default image.
+     * @throws IOException In case of error
+     */
+    @Test
+    @Ignore
+    public void deprecatesDefaultImage() throws IOException {
+        final Talk talk = new Talk.InFile();
+        FileUtils.write(
+            new File(this.start(talk, "").getAbsolutePath(), "testing"), "12"
+        );
+        new EndsDaemon().execute(talk);
+        for (final String path
+            : talk.read().xpath("/p/entry[@key='merge']/entry[@key='script']")
+        ) {
+            if ("yegor256/rultor".equals(path)) {
+                final String dir = talk.read()
+                    .xpath("/talk/daemon/dir/text()").get(0);
+                MatcherAssert.assertThat(
```

Comment:
@sebing just assert that it occurs twice: at the end and at the beginning.
Or maybe make assertion for entire deprecation text.